docs/guide/search/criteria_reference/maplocationdistance_criterion.md

```diff
@@ -0,0 +1,18 @@
+# MapLocationDistance Criterion
+
+The [`MapLocationDistance` Search Criterion](https://github.com/ezsystems/ezpublish-kernel/blob/v8.0.0-beta3/eZ/Publish/API/Repository/Values/Content/Query/Criterion/MapLocationDistance.php)
+searches content based on the distance between the location contained in a MapLocation Field and the provided coordinates.
+
+## Arguments
+
+- `target` - string representing the Field definition identifier
+- `operator` - Operator constant (IN, EQ, GT, GTE, LT, LTE, BETWEEN)
+- `distance` - float(s) representing the distances between the map location in the Field and the location provided in `latitude` and `longitude` arguments
```

Comment:
This needs to be more specific:
For operators `IN` and `BETWEEN` it's list of  `float`-s, for the other operators it's a single `float`.